.circleci/config.yml

```diff
@@ -103,13 +109,10 @@ commands:
     description: "Build Android app"
     steps:
       - checkout
-
       - restore_cache: *restore-npm-cache-linux
-
       - run: *install-npm-modules
-
-      - restore_cache: *restore-gradle-cache
-
+      - save_cache: *save-npm-cache-linux
+      # - restore_cache: *restore-gradle-cache
```

Comment:
Had to do it to build properly.
Removed here only, because e2e tests seems to work with it.
Probably because here it's bundling, while there it's assembling.